src/View/SSTemplateParser.peg

```diff
@@ -247,7 +238,7 @@ class SSTemplateParser extends Parser implements TemplateParser
         }
 
         $res['php'] .= ($sub['ArgumentMode'] == 'default') ? $sub['string_php'] :
-            str_replace('$$FINAL', 'XML_val', $sub['php'] ?? '');
+            str_replace('$$FINAL', 'getValueAsArgument', $sub['php'] ?? '');
```

Comment:
This allows us to define exactly how values are passed in as arguments for function calls in templates. It used to get the `string` representation of a value. See `SSViewer_Scope::getValueAsArgument()` for what is now returned.